ApplyToBecomeInternal/ApplyToBecome.Data/Models/Rationale.cs

```diff
@@ -0,0 +1,8 @@
+namespace ApplyToBecome.Data.Models
+{
+	public class Rationale
+	{
+		public string ProjectRationale { get; set; }
+		public string TrustRationale { get; set; }

```

Comment:
Would it be better to name these `RationaleForProject` and `RationaleForTrust` just to keep it consistent with the naming of the headings and the database columns

Reply:
Or something along those lines